tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java

```diff
@@ -264,12 +272,14 @@ public void run() {
         });
 
         try {
+            Random random = new Random(System.currentTimeMillis());
```

Comment:
Hmm, what's the reasoning for using this random seed? The default uses `System.nanoTime` combined with some other information. Also, I wonder if we should be printing the seed used?